pkg/3scale/amp/component/system.go

```diff
@@ -333,6 +335,8 @@ func (system *System) AddS3ConfigIntoEnvironmentConfigMap(configMap *v1.ConfigMa
 	configMap.Data["FILE_UPLOAD_STORAGE"] = "s3"
 	configMap.Data["AWS_BUCKET"] = system.Options.s3FileStorageOptions.AWSBucket
 	configMap.Data["AWS_REGION"] = system.Options.s3FileStorageOptions.AWSRegion
+	configMap.Data[StorageServiceEndpointScheme] = system.Options.s3FileStorageOptions.EndpointScheme
```

Comment:
We save the values of the scheme and host in the system-environment ConfigMap.
This is not really needed (could directly be referenced as an envvar) but for consistency with the other existing S3 fields (AWS_BUCKET, AWS_REGION), and to keep track of what's the value in every moment, and to have a centralized source of truth for the value I've decided to do it this way anyway